components/brave_wallet_ui/constants/types.ts

```diff
@@ -485,6 +489,7 @@ export interface GetAllTransactionInfoReturnInfo {
 export interface EthTxController {
   addUnapprovedTransaction: (txData: TxData, from: string) => Promise<AddUnapprovedTransactionReturnInfo>
   addUnapproved1559Transaction: (txData: TxData1559, from: string) => (AddUnapproved1559TransactionReturnInfo)
+  setGasPriceAndLimitForUnapprovedTransaction: (txMetaId: string, gasPrice: string, gasLimit: string) => Promise<SetGasPriceAndLimitForUnapprovedTransactionReturnInfo>
```

Comment:
EIP 1559 matching function to come in a different PR I assume right?

Reply:
yes correct